cmds/wifi/wifi.go

```diff
@@ -286,6 +294,8 @@ func main() {
 		return
 	}
 
+	// Start a Server with Stub data
+	// for manual end-to-end testing
```

Comment:
This just seems like you need two functions and the pointer suggested above?
Or ... make an interface with Scan and Connect and then have a Null implementation of Scan and Connect

This just doesn't seem right.

Reply:
Fixed